src/scheduler.ts

```
@@ -0,0 +1,64 @@
+export function clearSchedules() {
+  for (const t of timers) clearTimeout(t);
+  timers = [];
+}
+
+export function scheduleForActiveEvent(client: Client, guildId: string) {
+  clearSchedules();
+  const ev = repo.getActiveEvent(guildId);
+  if (!ev) return;
+  const settings = repo.getGuildSettings(guildId);
+  if (!settings.announcements_channel_id) return;
+  const channel = client.channels.cache.get(settings.announcements_channel_id) as
```

ESLint

[error] Replace `settings.announcements_channel_id)·as` with `⏎····settings.announcements_channel_id,`

(prettier/prettier)